dataframe-openapi-generator/src/main/kotlin/org/jetbrains/kotlinx/dataframe/io/utils.kt

```diff
@@ -18,7 +17,7 @@ internal fun String.withoutTopInterfaceName(topInterfaceName: ValidFieldName): S
         this
     }
 
-internal fun String.snakeToLowerCamelCase(): String = toCamelCaseByDelimiters(DELIMITERS_REGEX)
```

Comment:
Let's just deprecate this function or remove it

Reply:
I'm hesitating because it's used in the implementation, I don't understand why and how it should work so I'm just afraid of breaking something

Reply:
alright